docs/install/install-node.rst

```diff
@@ -58,9 +60,17 @@ Installing on CentOS 7
    .. task:: install_flocker centos-7
       :prompt: [root@centos]#
 
-#. **Repeat steps 1 and 2 for all other nodes:**
+#. **Install the** ``clusterhq-flocker-docker-plugin`` **package:**
+
+   Run the following command as root on the target node:
+
+   .. prompt:: bash [root@centos]#
+   
+      yum install -y clusterhq-flocker-docker-plugin
```

Comment:
Also, please file an issue to replace the explicit command with a task directive and to add to installation tests.